code/game/machinery/doors/airlock.dm

```diff
@@ -886,14 +886,11 @@ About the new airlock wires panel:
 	if(operating || welded || locked)
 		return
 	if(!forced)
-		//despite the name, this wire is for general door control.
-		//Bolts are already covered by the check for locked, above
 		if( !arePowerSystemsOn() || isWireCut(AIRLOCK_WIRE_OPEN_DOOR) )
 			return
 	if(safe)
-		for(var/turf/turf in locs)

```

Comment:
Uh, this seriously matters here- get_turf(src) won't return more than one turf, and we have multi-tile doors